reco_utils/dataset/movielens.py

```diff
@@ -136,12 +136,12 @@ def item_has_header(self):
 
 # Warning and error messages
 WARNING_MOVIE_LENS_HEADER = """MovieLens rating dataset has four columns
-    (user id, movie id, rating, and timestamp), but more than four column headers are provided.
-    Will only use the first four column headers."""
+    (user id, movie id, rating, and timestamp), but more than four column names are provided.
+    Will only use the first four column names."""
 WARNING_HAVE_SCHEMA_AND_HEADER = """Both schema and header are provided.
     The header argument will be ignored."""
 ERROR_MOVIE_LENS_SIZE = "Invalid data size. Should be one of {100k, 1m, 10m, or 20m}"
-ERROR_NO_HEADER = "No header (schema) information"
+ERROR_NO_HEADER = "No header (schema) information. At least user and movie column names should be provided"
```

Comment:
Really like these constant string of error/warning message. Do we later want to consolidate this into somewhere in `common/constants`? Or, at least, adopt this practice in other utility codes.

Reply:
If we need, sure why not. We can even generalize further, e.g. use `obj.__name__` or to accept names as args.